types/stylelint/index.d.mts

```diff
@@ -1,6 +1,7 @@
 export {
 	default,
 	Config,
+	ConfigRules,
```

Comment:
You are importing `ConfigRules` [here](https://github.com/stylelint/stylelint/pull/8297/files#diff-a885e25af8f2e8f600542d13e286eacdf8d80b1a527b58a0a6e88cc24063f7b5R19) but not using it.
If you are not planing to use it, can you revert this change and https://github.com/stylelint/stylelint/pull/8297/files#diff-9a4b261e295bef1ccce813aa194ac4023a1dcf98236667889964b1ba6fbc551fL11-L14?

Reply:
Good catch! Fixed in this commit.
[Refactor remove unused import](https://github.com/stylelint/stylelint/pull/8297/commits/0fbcbe41586facbde6e0ddfcf954f2c2ecc52bc3)

Reply:
You forgot to revert the changes in `index.d.ts`.